src/node/rpc/handler_registry.h

```diff
@@ -213,47 +208,46 @@ namespace ccf
         return *this;
       }
 
-      // Bit mask. Bit i is 1 iff the http_method with value i is allowed.
-      // Default is that all verbs are allowed
-      uint64_t allowed_verbs_mask = ~0;
+      http_method verb = HTTP_POST;
 
-      // https://github.com/microsoft/CCF/issues/1102
-      Handler& set_allowed_verbs(std::set<http_method>&& allowed_verbs)
+      /** Indicates which HTTP verb the handler should respond to.
+       *
+       * @return The installed Handler for further modification
+       */
+      [[deprecated("HTTP Verb should not be changed after installation: pass verb to install()")]] Handler&
```

Comment:
For discussion: Is it worth listing these as deprecated for a release? It means the API change is just a warning rather than a compile error, and points clearly to the intended resolution. On the other hand, it ties us to supporting old APIs for longer, and we need to remember to go delete our deprecations at some point.

I'm thinking this instance is a useful test run, and we should get in the habit of deprecating-then-deleting moving forward, as we'll need to post-1.0.